integration-tests/smoke-tests/pom.xml

```
@@ -6,13 +6,15 @@
     <parent>
         <groupId>com.github.mcollovati</groupId>
         <artifactId>quarkus-hilla-parent</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>${revision}</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
 
     <artifactId>quarkus-hilla-smoke-tests</artifactId>
+    <name>Quarkus - Hilla - Integration Tests</name>
```

```suggestion
    <name>Quarkus - Hilla - Smoke Tests</name>
```
Matching the artifactId, even if these are currently all tests we have.